recipes/libusrsctp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```

Avoid to include headers folder, so that include property of imported target is really tested.

---

Thanks, good to know